src/templates/build-release/build-release-android.ejf

```diff
@@ -49,6 +49,12 @@ jobs:
       - name: 🏗 Checkout repository
         uses: actions/checkout@v4
 
+      - name: 🌿 Setup Node 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
```

Comment:
I wonder if we shouldn't guess the node version somehow instead of hardcoding it

Reply:
added dynamic determination of node version in order respecting https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file:
1. `.nvmrc` file in **package root**
2. `.node-version` file in **package root** 
3. `engines.node` or `volta.node` field in `package.json` in **package root** 
4. 1. `.nvmrc` file in **repo root**
5. `.node-version` file in **repo root**
6. `engines.node` or `volta.node` field in `package.json` in **repo root**

and if 1-6 fail, create `.nvmrc` with `v20.17.0` in **package root**

then we use the path to node version file in `Setup node` step with field `node-version-file`